### ISSUE TITLE
Default theme colors are inaccessible (links, sidebar text)

### ISSUE THREAD

**Issue by alexkrolick** [facebook/docusaurus#4947]
Previous: https://github.com/facebook/docusaurus/issues/2484#issuecomment-754886515

## Links

Link/Primary color: `#25c2a0` fails WCAG at every size (contrast ratio [2.26](https://contrastchecker.com/?c=25c2a0&b=ffffff
) against white)

<img width="663" alt="Screen Shot 2021-01-05 at 12 25 26 PM" src="https://user-images.githubusercontent.com/1571667/103696034-84459f80-4f52-11eb-9b6c-949403c23f2e.png">

## Sidebar

Sidebar/Secondary color: `#606770` [marginal](https://contrastchecker.com/?c=606770&b=ffffff) at smaller sizes

<img width="220" alt="Screen Shot 2021-01-05 at 12 25 31 PM" src="https://user-images.githubusercontent.com/1571667/103695643-d4703200-4f51-11eb-8a0d-85864a65799b.png">

**Comment by Josh-Cena** [comment]
![image](https://user-images.githubusercontent.com/55398995/129205050-67fcbe25-e8d4-4d2c-9c8c-c585d98938b8.png)

As lighthouse reports, the buttons on the frontpage are also inaccessible. But this only applies to the website anyways and doesn't harm the theme itself
